interface/forms/bronchitis/table.sql

```diff
@@ -2,90 +2,90 @@ CREATE TABLE IF NOT EXISTS `form_bronchitis` (
 id bigint(20) NOT NULL auto_increment,
 date datetime default NULL,
 pid bigint(20) default NULL,
-user varchar(255) default NULL,
-groupname varchar(255) default NULL,
+user varchar(50),
+groupname varchar(50),
```

Comment:
dang, sorry, i forgot to add that would keep the default NULL there since it was there to begin with and keeping it a varchar :pray:

Reply:
good pt, no prob, thank you